service/templates/projects.html

```diff
@@ -296,24 +296,21 @@ <h2> Settings for [[displayName]]</h2>
           </div>
         </div>
 
-        <div class="variables" ng-show="isSuperUser">
+        <div class="variables">
           <div class="form-group">
-            <div class="col-header col-lg-6 form-control-static"> <h3> GitHub Webhook Installation </h3>
-                <div class="muted small" ng-show="githubHookID === 0">Github webhooks have not been enabled for this repository. In order to use Github Pull Request testing and the commit queue, tick the box below, and save the project to set up the webhook in this
-                repository.</div>
-                <div class="muted small" ng-show="githubHookID !== 0">Github webhooks are enabled!</div>
-            </div>
-          </div>
-
-          <div id="patch-variants-list" class="form-group">
-            <div class="col-lg-6">
-                <input type="checkbox" id="githubhook-checkbox" ng-disabled="githubHookID !== 0" ng-model="settingsFormData.setup_github_hook" />
-                <label for="githubhook-checkbox">Enable Github Webhooks</label>
+            <div class="col-header col-lg-6 form-control-static"> <h3> GitHub Webhooks </h3>
+                <div ng-show="scope.github_webhooks_enabled">
+                  GitHub webhooks are enabled.
+                </div>
+                <div class="project-error" ng-show="!scope.github_webhooks_enabled">
```

Comment:
Won't this always show for new projects?

Reply:
I added a flag for new projects so until the first save the user won't see an error.
They will see it if they reload.

Reply:
The overall approach seems questionable to me. Why not just display a message saying that web hooks aren't enabled?

Reply:
@bsamek suggested making the message appear more urgent to the user so they will seek help.

Reply:
I think we could have some middle ground where it appears urgent but also a user will understand if it appears on their new project. How about "GitHub Pull Request testing and the commit queue are disabled because web hooks are not enabled. Web hooks are enabled after entering a repository and branch."

Reply:
done.